src/models/foreignBornDocument.js

```diff
@@ -18,9 +18,12 @@ const foreignBornDocument = {
 
     return {}
   },
-  DocumentExpiration: (value, attributes) => {
-    if (attributes.DocumentExpirationNotApplicable
-      && attributes.DocumentExpirationNotApplicable.applicable === false) {
+  DocumentExpiration: (value, attributes, attributeName, options) => {
+    if (
+      (attributes.DocumentExpirationNotApplicable
+      && attributes.DocumentExpirationNotApplicable.applicable === false)
+      || !options.requireRelationshipMaritalForeignBornDocExpiration
```

Comment:
can we name options independently of the context from where they're being set (i.e. `requireForeignBornDocExpiration`)?